### DISCUSSION TITLE
Is it possible to read/write blob/binary data?

### DISCUSSION THREAD

**Discussion by wajihullahbaig** [BlueBrain/HighFive#484] [General]
I was looking through the examples and could not find any examples that would show as to how blob/binary data can be read and written. So I was wondering if it possible and what would be the correct way of doing it. 
Any such examples would be really helpful. 

Thanks!

**Comment by ohm314** [comment]
Sorry, clearly this didn't get answered for a very long time. But yes, you can of course write arrays of `char`, which is exactly that. If you don't want to encode any structure of the data into HDF5, you don't need to. A one-dimensional data set would do.

**Reply by wajihullahbaig** [reply]
Thanks for the reply. That is actually what I did in the end :)
